File: bealquileres/src/main/java/edu/uclm/esi/tysweb/bealquileres/dto/VehiculoDto.java
package edu.uclm.esi.tysweb.bealquileres.dto;

/**
 * VehiculoDto
 */
public record VehiculoDto(Long id, String matricula, Double latitudActual, Double longitudActual,
    Double distanciaTotal, Double bateria, Boolean activo, String municipio) {

}

File: feayto/src/app/model/VehiculoDto.ts
export interface VehiculoDto {
  id : number
  matricula : string
  latitudOrigen : number
  longitudOrigen : number
  nombreOrigen : string
  latitudDestino : number
  longitudDestino : number
  nombreDestino : string
  latitudActual : number
  longitudActual : number
  direccion : string
  kmh : number
  distanciaRecorrida : number
  distanciaTotal : number
  bateria : number
  activo : boolean
  municipio : string
}